scripts/myvectorbench.py

```
@@ -0,0 +1,641 @@
+    ef = wp.get('ef_construction', 200)
+    rows = len(vectors)
+    print(f"  [insert_throughput] {rows} rows, dim={dim}, online=Y")
+
+    _create_bench_table(container, dim, rows, M, ef, "bench", "insert_t", online=True)
+
+    t0 = time.time()
+    batch = 500
+    for start in range(0, rows, batch):
+        chunk = vectors[start:start + batch]
+        vals = ", ".join(f"({start + i}, {_vec_literal(v)})" for i, v in enumerate(chunk))
+        container.sql(f"INSERT INTO bench.insert_t (id, vec) VALUES {vals};")
```

Ruff (0.15.13)

[error] Possible SQL injection vector through string-based query construction

(S608)